README.md

```diff
@@ -14,20 +14,22 @@
         - With gcloud, to create cluster `gcloud container clusters create [$CLUSTER_NAME] --project [$PROJECT_NAME] --[region/zone] [$REGION/ZONE]`
 
         - To generate kubeconfig   `gcloud container clusters get-credentials [$CLUSTER_NAME] --project [$PROJECT_NAME] --[region/zone] [$REGION/ZONE]`
-3. Create an fsi namespace and enable istio-injection
+3. (Optional) Apply Istio manifests to cluster for **local** development
+   - Run `istioctl manifest apply`
+4. Create an fsi namespace and enable istio-injection
    - Run `kubectl create namespace fsi | kubectl label namespace fsi istio-injection=enabled` to create the fsi namespace and enable istio sidecar auto-injection for this namespace
-4. Configure service account secret for the services to use.
+5. Configure service account secret for the services to use.
    - Run `kubectl create secret generic svc-key -n fsi --from-file [$ABSOLUTE_PATH_TO_KEY]/service-acc.json` to make a k8s secret to store credentials
-5. Configure project id as a configmap for the services to use.
+6. Configure project id as a configmap for the services to use.
    - Run `kubectl create configmap env-vars -n fsi --from-literal=env.project-id=[$PROJECT_ID] -o yaml --dry-run | kubectl apply -f -` to make a k8s configmap with project id
-6. Build and run services by following any one of the steps below.
+7. Build and run services by following any one of the steps below.
    - Run `skaffold run` for running locally
    - Run `skaffold run --default-repo=gcr.io/[$PROJECT_ID]` for getting images from remote registry
    - RUN `skaffold run -p gcb --default-repo=gcr.io/[$PROJECT_ID]` for remotely building docker images
    - Run `skaffold dev --default-repo=gcr.io/[$PROJECT_ID]` for constantly watching code changes locally and rebuilding locally
    - Run `skaffold dev -p gcb --default-repo=gcr.io/[$PROJECT_ID]` for constantly watching code changes locally and rebuilding remotely
 
-7. Application should be available at istio-gateway external IP or localhost
+8. Application should be available at istio-gateway external IP or localhost
    - Frontend: localhost/istio-ingressgateway-ip:80
```

Comment:
Can you add the other API endpoints here as well

Reply:
Updated ReadMe